### PR TITLE
Parallelize Travis build (closes #50)

### LINKED ISSUE #50: Parallelize Travis build
Instead of using a custom script (`sbt ... +test`) with sbt's flaky cross compilation support by `+`, better define multiple Scala versions in the Travis configuration and use `sbt_args`. See the [Travis docs](https://docs.travis-ci.com/user/languages/scala). This will spawn parallel builds for each Scala version.

### PR DESCRIPTION
**Pull request by hseeberger**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ sudo: required
 
 scala:
   - "2.11.8"
+  - "2.12.0"
```
Comment (ktoso) [inline .travis.yml]: Shouldn't this also change the `+test` to `test` in this script then though?

Comment (hseeberger) [inline .travis.yml]: Yes. I deleted the custom script. Travis applies the following default script: `sbt ++$TRAVIS_SCALA_VERSION test`. All schould be fine.

Comment (ktoso) [inline .travis.yml]: Cool 👍

**Comment by ktoso** [comment]
LGTM, thanks!

**Comment by ktoso** [comment]
resolves #50

**Comment by hseeberger** [comment]
Look Ma, two parallel builds: https://travis-ci.org/akka/alpakka/builds/175448162

**Comment by ktoso** [comment]
UNCOMFORTABLY PARALLEL https://youtu.be/1yH_j8-VVLo?t=12s 😜

**Comment by patriknw** [comment]
How does travis work. Will these two jobs run in parallel and compete for the same (very limited) machine resources?

**Comment by hseeberger** [comment]
Phew ... I don't know the details. But it looks like they run in parallel.

**Pull request merged**